webpack.config.js

```diff
@@ -0,0 +1,128 @@
+/* --------------------------------------------------------------------
+ * Fill in module info here.
+ */
+
+var entries = {
+    "sosreport/sosreport": [
+        "sosreport/index.js",
+        "sosreport/sosreport.css",
+    ],
+    "subscriptions/subscriptions": [
+        "subscriptions/main.js",
+        "subscriptions/subscriptions.css",
+    ],
+    "users/users": [
+        "users/local.js",
+        "users/users.css",
+    ]
+};
+
+var files = [
+    "sosreport/index.html",
+    "sosreport/sosreport.png",
+    "sosreport/manifest.json",
+
+    "users/index.html",
+    "users/manifest.json",
+];
+
+var externals = {
+    "jquery": "$",
+    "cockpit": "cockpit",
+};
+
+/* ---------------------------------------------------------------------
+ * Implementation
+ */
+
+var webpack = require("webpack");
+var copy = require("copy-webpack-plugin");
+var extract = require("extract-text-webpack-plugin");
+var path = require("path");
+
+/* For node 0.10.x we need this defined */
+if (typeof(global.Promise) == "undefined")
+    global.Promise = require('promise');
+
+/* These can be overridden, typically from the Makefile.am */
+var srcdir = process.env.SRCDIR || __dirname;
+var pkgdir = path.join(srcdir, "pkg")
+var distdir = path.join(process.env.BUILDDIR || __dirname, "dist");
+
+/* Qualify all the paths in entries */
+Object.keys(entries).forEach(function(key) {
+    entries[key] = entries[key].map(function(value) {
+        return path.join(pkgdir, value);
+    });
+});
+
+/* Qualify all the paths in files listed */
+files = files.map(function(value) {
+    return { from: path.join(pkgdir, value), to: value };
+});
+
+module.exports = {
+    resolve: {
+        alias: {
+<<<<<<< HEAD
+            "mustache": "mustache/mustache.js",
+=======
+            mustache: "mustache/mustache.js",
+            react: "react-lite-cockpit/dist/react-lite.js",
+>>>>>>> d16f903... subscriptions: Migrate to webpack
```

Comment:
is this leftover from a diff/merge?
also subscriptions don't use react currently